client/src/main/kotlin/io/spine/examples/pingh/client/DesktopClient.kt

```diff
@@ -167,10 +174,16 @@ public class DesktopClient(
     }
 
     /**
-     * Marks the mention as a 2-hour snooze.
+     * Marks the mention as snoozed.
+     *
+     * If snooze time is not specified, the mention will snooze the `defaultSnoozeTime`.
```

Comment:
How a reader of this piece is going to know where to find this `defaultSnoozeTime`? I would at least have a reference here. Not sure how it is going to render, given the field is `private` inside the corresponding companion object, though.